python/mxnet/contrib/text/__init__.py

```diff
@@ -19,6 +19,5 @@
 """This module includes utilities for indexing and embedding text."""
 
 from . import utils
-from . import indexer
+from . import vocab
```

Comment:
Since `Vocabulary` is the only class that's in that namespace, we can consider removing this namespace by importing its members.

Reply:
This is intended for potential future extension.

Reply:
Do you have such cases in mind? `text.vocab.Vocabulary` looks less clean than `text.Vocabulary`, and `vocab` is neither a proper word nor common enough to become an expected package name.

Reply:
Thanks. For instance, SubwordVocabulary may be added in the future.

Reply:
I think this only depends on tokenization. The class of vocabulary itself doesn't need any change.